features/providers/back_button.feature

```diff
@@ -0,0 +1,214 @@
+#  The test below checks the back button functionality across multiple pages
+#  Introduced as a check after issues with the the use of the back button were identified
```

Comment:
Should this file be commented out?

Reply:
Yes. These are the tests that revolve around the use of the back button, they were written when we had issues with back button behaviour. We fixed that so with the ethos of feature tests only testing happy paths these can be commented out with the option to easily re-enable for testing in the future. I dont feel they need to be run every time but they have value as regression tests if we experience back button issues in the future.

Reply:
👍